ci/unit/command.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This is intended to be run inside the docker container as the command of the docker-compose.
+set -ex
+
+export PATH=$PATH:$PWD/vendor/jruby/bin
+gem install bundler
+cd ../plugins/this
+cp /usr/share/logstash/logstash-core/versions-gem-copy.yml /usr/share/logstash/logstash-core-plugin-api/versions-gem-copy.yml
+#./gradlew vendor
+bundle install
+bundle exec rspec -fd --pattern spec/**/*_spec.rb,spec/**/*_specs.rb
```

Comment:
why the need to include `_specs.rb`? we could just have `bundle exec rspec -fd`

Reply:
the default pattern is '"**{,/*/**}/*_spec.rb"' , and i _think_ i saw (wrote?) some tests that end in _specs. Also if we introduce more integration specs they are less likely to get picked up by the default glob.